data_managers/data_manager_eukcc/data_manager/eukcc.xml

```diff
@@ -0,0 +1,59 @@
+<tool id="data_manager_eukcc" name="EukCC data manager" version="@TOOL_VERSION@+galaxy@VERSION_SUFFIX@" tool_type="manage_data" profile="20.01">
+    <macros>
+        <token name="@TOOL_VERSION@">2.1.3</token>
+        <token name="@VERSION_SUFFIX@">0</token>
+    </macros>
+    <requirements>
+        <requirement type="package" version="@TOOL_VERSION@">eukcc</requirement>
+    </requirements>
+    <version_command>eukcc --version | cut -d" " -f3</version_command>
+    <command detect_errors="exit_code"><![CDATA[
+        mkdir '$output_file.extra_files_path' &&
+        wget http://ftp.ebi.ac.uk/pub/databases/metagenomics/eukcc/eukcc2_db_ver_${version}.tar.gz &&
+        tar -xvf eukcc2_db_ver_${version}.tar.gz &&
+        rm eukcc2_db_ver_${version}.tar.gz &&
+        mv eukcc2_db_ver_${version}/* '$output_file.extra_files_path'/ &&
+        cp '$dmjson' '$output_file'
+    ]]></command>
+    <configfiles>
+        <configfile name="dmjson"><![CDATA[
+    {
+      "data_tables":{
+        "eukcc":[
+          {
+            "value":"$version",
+            "name":"EukCC DB ${version}",
+            "version":"${version}",
+            "path":"$output_file.extra_files_path"
+          }
+        ]
+      }
+    }]]></configfile>
+    </configfiles>
+    <inputs>
+        <param name="version" type="select" label="DB version">
+            <option value="1.2">1.2</option>
```

Comment:
Should this here have a editable version as well.

E.g. make a param="text" with "options".

Reply:
I would perfer to add a new version if needed, i.e. if new reference data is released.

Reply:
Good point. I will still leave it as it is for now since I do not have the time to deal with quoting, validators and thinking about the use of the glob (*) in the command block.